index.html

```diff
@@ -361,6 +361,14 @@ <h3>P-521</h2>
         </pre>
       </section>
 
+      <section class="informative">
+        <h3>RSA</h3>
```

Comment:
I would prefer to see examples start with RSA 2048 and end with RSA 4096, I expect folks to ROFL at the 4096 did document.

Reply:
Added one for 4096. Look out for ROFLcopters!

I put it in a separate section because the ASCII prefix for the multicodec value is different depending on the length of the payload, apparently.